exporters/trace/jaeger/env.go

```diff
@@ -15,31 +15,21 @@
 package jaeger // import "go.opentelemetry.io/otel/exporters/trace/jaeger"
 
 import (
-	"errors"
 	"os"
 	"strconv"
-	"strings"
-
-	"go.opentelemetry.io/otel"
-	"go.opentelemetry.io/otel/attribute"
 )
 
 // Environment variable names
 const (
-	// The service name.
-	envServiceName = "JAEGER_SERVICE_NAME"
 	// Whether the exporter is disabled or not. (default false).
-	envDisabled = "JAEGER_DISABLED"
-	// A comma separated list of name=value tracer-level tags, which get added to all reported spans.
-	// The value can also refer to an environment variable using the format ${envVarName:defaultValue}.
-	envTags = "JAEGER_TAGS"
+	envDisabled = "OTEL_EXPORTER_JAEGER_DISABLED"
```

Comment:
We should remove `OTEL_EXPORTER_JAEGER_DISABLED` since the specification does not define this env anymore.